source/guides/azure/infiniband.md

```diff
@@ -0,0 +1,493 @@
+```
+
+Reboot and reconnect.
+
+```shell
+sudo reboot
+```
+
+## Check IB
+
+```shell
+ip addr show
+```
+
+```shell
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo
+       valid_lft forever preferred_lft forever
+    inet6 ::1/128 scope host
+       valid_lft forever preferred_lft forever
+2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
+    link/ether 60:45:bd:a7:42:cc brd ff:ff:ff:ff:ff:ff
+    inet 10.6.0.5/24 brd 10.6.0.255 scope global eth0
+       valid_lft forever preferred_lft forever
+    inet6 fe80::6245:bdff:fea7:42cc/64 scope link
+       valid_lft forever preferred_lft forever
+3: eth1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
+    link/ether 00:15:5d:33:ff:16 brd ff:ff:ff:ff:ff:ff
+4: enP44906s1: <BROADCAST,MULTICAST,SLAVE,UP,LOWER_UP> mtu 1500 qdisc mq master eth0 state UP group default qlen 1000
+    link/ether 60:45:bd:a7:42:cc brd ff:ff:ff:ff:ff:ff
+    altname enP44906p0s2
+5: ibP59423s2: <BROADCAST,MULTICAST> mtu 4092 qdisc noop state DOWN group default qlen 256
+    link/infiniband 00:00:09:27:fe:80:00:00:00:00:00:00:00:15:5d:ff:fd:33:ff:16 brd 00:ff:ff:ff:ff:12:40:1b:80:1d:00:00:00:00:00:00:ff:ff:ff:ff
+    altname ibP59423p0s2
+```
+
+```shell
+nvidia-smi topo -m
+```
+
+```shell
+        GPU0    GPU1    GPU2    GPU3    GPU4    GPU5    GPU6    GPU7    CPU Affinity    NUMA Affinity
+GPU0    X       NV2     NV1     NV2     NODE    NODE    NV1     NODE    0-19            0
+GPU1    NV2     X       NV2     NV1     NODE    NODE    NODE    NV1     0-19            0
+GPU2    NV1     NV2     X       NV1     NV2     NODE    NODE    NODE    0-19            0
+GPU3    NV2     NV1     NV1     X       NODE    NV2     NODE    NODE    0-19            0
+GPU4    NODE    NODE    NV2     NODE    X       NV1     NV1     NV2     0-19            0
+GPU5    NODE    NODE    NODE    NV2     NV1     X       NV2     NV1     0-19            0
+GPU6    NV1     NODE    NODE    NODE    NV1     NV2     X       NV2     0-19            0
+GPU7    NODE    NV1     NODE    NODE    NV2     NV1     NV2     X       0-19            0
+
+Legend:
+
+  X    = Self
+  SYS  = Connection traversing PCIe as well as the SMP interconnect between NUMA nodes (e.g., QPI/UPI)
+  NODE = Connection traversing PCIe as well as the interconnect between PCIe Host Bridges within a NUMA node
+  PHB  = Connection traversing PCIe as well as a PCIe Host Bridge (typically the CPU)
+  PXB  = Connection traversing multiple PCIe bridges (without traversing the PCIe Host Bridge)
+  PIX  = Connection traversing at most a single PCIe bridge
+  NV#  = Connection traversing a bonded set of # NVLinks
+
+```
+
+## Install UCX-Py and tools
+
+```shell
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh
```

Comment:
Not sure if you want to make sure users read EULA and accept, but otherwise you could add `-b` here:

```
-b           run install in batch mode (without manual intervention),
             it is expected the license terms are agreed upon
```